service/controller/v13/resource/instance/create_deployment_completed.go

```diff
@@ -0,0 +1,83 @@
+package instance
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/azure-operator/service/controller/v13/blobclient"
+	"github.com/giantswarm/azure-operator/service/controller/v13/key"
+	"github.com/giantswarm/azure-operator/service/controller/v13/resource/instance/internal/state"
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) deploymentCompletedTransition(ctx context.Context, obj interface{}, currentState state.State) (state.State, error) {
+	customObject, err := key.ToCustomObject(obj)
```

Comment:
SOTA says it is `cr`.

Reply:
I know, but just like many other things such as those log entries, I didn't want to change too much at once. There's definitely bunch of boyscout targets after this.